src/pybind/mgr/influx/module.py

```diff
@@ -82,7 +83,7 @@ def get_daemon_stats(self):
                 value = counter_info['value']
 
                 data.append({
-                    "measurement": "ceph_osd_stats",
```

Comment:
This change appears to be unrelated to the commit message -- if it's intentional, please separate it out into a different commit.